src/containers/MyNdla/userMutations.ts

```diff
@@ -61,8 +62,14 @@ export const usePersonalData = () => {
 };
 
 const updatePersonalDataQuery = gql`
-  mutation updatePersonalData($favoriteSubjects: [String!]!) {
-    updatePersonalData(favoriteSubjects: $favoriteSubjects) {
+  mutation updatePersonalData(
+    $favoriteSubjects: [String!]
```

Comment:
Denne blei slik i graph-skjema. Veit ikkje om det har nokon effekt.
```suggestion
    $favoriteSubjects: [String]
```